cli/run.go

```diff
@@ -13,7 +13,6 @@ import (
 )
 
 type RunCmd struct {
-	target      string
```

Comment:
We do use a `target` in runtime-context...

Reply:
Acknowledged, but it's not currently in-use in this branch.